src/js/utils/helper.js

```diff
@@ -0,0 +1,450 @@
+  }
+  return shuffled;
+};
+
+const updateSoundList = () => {
+  const soundsList = GET_VAR('soundsList').slice();
+  soundsList.pop();
+  SET_VAR('soundsList', soundsList);
+};
+
+const createNewSoundsList = () => {
+  const cards = document.querySelectorAll('.card-list__item');
+  const soundsList = [];
+  cards.forEach((card) => {
+    const { sound } = card.dataset;
+    soundsList.push(sound);
+  });
+
+  return soundsList;
+};
+
+const percentCorrectAnswers = (obj) => Math.round((
+  obj.Correct / (obj.Correct + obj.Incorrect)) * 100);
+
+const createStartingStatisticData = (data) => {
+  const result = [];
+  data.forEach((item) => {
+    if (item.section !== 'home') {
+      item.sectionWords.forEach((elem) => {
+        const obj = {
+          Category: item.sectionTitle,
+          Word: elem.word,
+          Translation: elem.translation,
+          Trained: 0,
+          Correct: 0,
+          Incorrect: 0,
+          'Accuracy %': 'n/e',
+        };
+        result.push(obj);
+      });
+    }
+  });
+  return result;
+};
+
+const createStatisticsData = (data) => {
+  const result = [];
+  const savedData = localStorage.getItem('statisticData');
+  if (!savedData) {
+    const startingData = createStartingStatisticData(data);
+    const startingDataToJson = JSON.stringify(startingData);
+    localStorage.setItem('statisticData', startingDataToJson);
+    result.push(...startingData);
+  } else {
+    const currentData = JSON.parse(savedData);
+    result.push(...currentData);
+  }
+
+  return result;
+};
+
+const capitalizeFirstWord = (word) => word.charAt(0).toUpperCase() + word.slice(1);
+
+const setGlobalValues = () => {
+  SET_VAR('mainSection', document.querySelector('.main'));
+  SET_VAR('categoryName', document.querySelector('.category-name'));
+  SET_VAR('cardsList', document.querySelector('.cards-list'));
+  SET_VAR('hamburgerBtn', document.querySelector('.header-hamburger'));
+  SET_VAR('headerNav', document.querySelector('.header-nav'));
+  SET_VAR('headerMenu', document.querySelector('.header-menu'));
+  SET_VAR('navMenuLinks', document.querySelectorAll('.header-menu__item-link'));
+  SET_VAR('overlay', document.querySelector('.overlay'));
+  SET_VAR('body', document.querySelector('body'));
+  SET_VAR('headerLogo', document.querySelector('.header-logo__link'));
+  SET_VAR('toggleBtn', document.querySelector('.toggle-button'));
+  SET_VAR('playRepeatBtn', document.querySelector('.play-btn'));
+  SET_VAR('scoreLine', document.querySelector('.score-line'));
+  SET_VAR('modalWindow', document.querySelector('.modal-window'));
+  SET_VAR('modalWindowText', document.querySelector('.modal-window__text'));
+  SET_VAR('modalWindowIcon', document.querySelector('.modal-window__icon'));
+  SET_VAR('statisticData', createStatisticsData(dataSet));
+  SET_VAR('isPlayMode', GET_VAR('toggleBtn').ariaPressed);
+};
+
+const resetCardsState = () => {
+  const cards = document.querySelectorAll('.card-front');
```

Comment:
it's better to cache the variable, because each time you invoke the function - the function will search for a DOM node again.